prow/plugins/config.go

```diff
@@ -39,8 +39,7 @@ const (
 type Configuration struct {
 	// Plugins is a map of repositories (eg "k/k") to lists of
 	// plugin names.
-	// TODO: Link to the list of supported plugins.
-	// https://github.com/kubernetes/test-infra/issues/3476
+	// You can find comprehensive list of plugins here https://prow.k8s.io/command-help
```

Comment:
Wouldn't https://github.com/kubernetes/test-infra/tree/master/prow/plugins be a better link?

Also note that there can be external plugins as well

Reply:
I agree it would provide more information, I got the link from #3476